README.md

```diff
@@ -53,7 +53,7 @@ local function TestOneInput(buf)
     end
 end
 
-luzer.Fuzz(TestOneInput)
+luzer.Fuzz(TestOneInput, nil, arg)
```

Comment:
`arg` is added, but in a step 3 it is unused (no options are passed to a script).

Reply:
Fixed it